modules/tinymce/src/core/main/ts/api/EditorObservable.ts

```diff
@@ -137,6 +137,11 @@ const bindEventDelegate = function (editor: Editor, eventName: string) {
   }
 };
 
+// EditorEventMap does not strictly inherit from NativeEventMap, which makes it
```

Comment:
This is incorrect. As you'll see in the `develop` branch both `Observable` and `EventDispatcher` have been updated so it doesn't need to extend `NativeEventMap`. We should backport that to here as well.

- [ ] Remove ts-ignore and backport fixes to `Observable` and `EventDispatcher` generics.